package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "pxt-rovercode",
+  "version": "0.0.1",
+  "description": "Rovercode PXT service for micro:bit",
+  "main": "/main.ts",
+  "repository": "https://github.com/rovercode/rovercode-pxt",
+  "author": "Brady Hurlburt",
+  "license": "GPL-3.0",
+  "private": false,
+  "scripts": {
+    "postinstall": "node_modules/pxt/pxt target microbit && pxt install",
```

Comment:
Why is the path needed?

Reply:
I tried the `pxt` command manually after the install, and it couldn't find it without the path. Is that not usually the case?

Reply:
Normally, no. You can see that all these don't: https://github.com/rovercode/rovercode-ui/blob/alpha/package.json#L11
Maybe something is wrong with the `pxt` package

Reply:
Woof, this killed the Travis build, too, even though I didn't change anything in the travis.yml. I guess it automatically does an `npm install` if it sees a package.json.

Reply:
Ok, got the travis build fixed -- I had a typo on the post install.
I'm still not sure why the `node_modules` path is needed, but it was the same on Travis.